src/jit/assertionprop.cpp

```diff
@@ -320,17 +321,17 @@ void Compiler::optAddCopies()
 #endif
 
             /* We have already calculated paramImportantUseDom above. */
-
-            BLOCKSET_ITER_INIT(this, iter, paramImportantUseDom, blkNum);
-            while (iter.NextElem(&blkNum))
+            BlockSetOps::Iter iter(this, paramImportantUseDom);
+            unsigned          bbNum = 0;
+            while (iter.NextElem(&bbNum))
```

Comment:
and is it better to use
```
BlockSetOps::Iter iter(this, paramImportantUseDom); 
for (unsigned bbNum; iter.NextElem(&bbNum); )
{
```
because we do not use the init value.